moon/controller_round1.py

```diff
@@ -7,6 +7,8 @@
 
 from datetime import timedelta
 from functools import partial
+from shapely.geometry import Point, Polygon
```

Comment:
I am not sure if to comment stupid details like this ... but for info ... the imports are recommended to be grouped into first Python native, empty line, 3rd party, empty line, internal (PEP8). But feel free to ignore.
(here `shapely` is 3rd party and `collections` is native Python)